src/Incidents/components/IncidentTypes/ActionBar/index.js

```diff
@@ -0,0 +1,92 @@
+import { getIncidentTypes } from '@codetanzania/emis-api-states';
+import { Button, Checkbox, Col, Pagination, Row } from 'antd';
+import PropTypes from 'prop-types';
+import React from 'react';
+import './styles.css';
+
+/**
+ * Render action bar for actions which are applicable to list content
+ *
+ * @function
+ * @name IncidentTypesActionBar
+ *
+ * @param {Object} props
+ * @param {page} props.page
+ * @param {number} props.total
+ *
+ * @version 0.1.0
+ * @since 0.1.0
+ */
+const IncidentTypesActionBar = ({ page, total }) => (
+  <div className="IncidentTypesActionBar">
+    <Row>
+      <Col span={1} xl={1} className="checkbox">
+        <Checkbox />
+      </Col>
+
+      <Col span={1} xl={1}>
+        <Button
+          shape="circle"
+          icon="reload"
+          title="Refresh incidenttypes"
+          onClick={() => getIncidentTypes()}
+          className="actionButton"
+          size="large"
+        />
+      </Col>
+
+      <Col span={1} xl={1}>
+        <Button
+          type="circle"
+          icon="cloud-download"
+          title="Export selected incidenttypes"
```

Comment:
This is a title supposed to be `Export selected Incident Types`

Reply:
So for other places for title attribute change `incidenttypes`  to be a normal word

Reply:
oky will fix that